tests/conftest.py

```diff
@@ -312,20 +318,18 @@ def _subprocess_setup(request, cfg=None, uwsgi=False):
     from pykern import pkunit
     from pykern import pkio
 
-    # different port than default so can run tests when supervisor running
-    p = "8101"
     cfg.pkupdate(
         PYKERN_PKDEBUG_WANT_PID_TIME="1",
         SIREPO_PKCLI_JOB_SUPERVISOR_IP=_LOCALHOST,
```

Comment:
All services should use _LOCALHOST so set  SIREPO_PKCLI_SERVICE_IP=_LOCALHOST.

Reply:
[61c6ce8](https://github.com/radiasoft/sirepo/pull/4846/commits/61c6ce81f04d1b0e1421ea52ca338c45c93bd8f3)